chart/templates/configmap.yaml

```diff
@@ -84,21 +84,23 @@ data:
         {{ $key }}: {{ $val | quote }}
         {{- end }}
       {{- end }}
+      {{- if .Values.injector.resources }}
```

Comment:
```suggestion
      {{- if .Values.injector.tolerations }}
```
why resources?

Reply:
You are to fast !
Copilot auto complete, i noticed after committing lol